docker/pypi/wmagent/wmagent-docker-run.sh

```diff
@@ -0,0 +1,73 @@
+#!/bin/bash
+
+### This script is to be used for running the WMAgent docker container at a VM
+### Its sole purpose is to set all the needed mount points from the Host VM and
+### forward all WMAgent runtime parameters to the WMAgent container entrypoint run.sh
+### It accepts only the set of parameters relevant to WMAgent's container run.sh
+### and no build dependent ones. The docker image tag to be searched for execution is
+### always `latest`.
+
+
+# NOTE: In the help call to the current scrit we only repeat the help and usage
+#       information for all the parameters accepted by run.sh.
+help(){
+    echo -e $1
+    cat <<EOF
+
+The script to be used for running a WMAgent docker container at a VM. The full set of arguments
+passed to the current script are to be forwarded to the WMAgent container entrypoint 'run.sh'
+
+Usage: wmagent-docker-run.sh [-t <team_name>] [-n <agent_number>] [-c <central_services_url>] [-f <db_flavour>]
+
```

Comment:
It should support the WMAgent tag that we want to pull/run as well. Otherwise it will run whatever image version we have in the host (lacking it, I guess it would pull the latest tag).

Reply:
As I said it in several places already, it is always a local build, so for the time being it will always be pointing to the `latest` version. And it is actually intentional. As you can see we are explicitly building against two tags `$WMA_TAG` and `latest`. This is a crucial part of the protection for one not to be able to run two `wamagent` containers on the same host.

Reply:
Similarly to the `wmagent-docker-build.sh` a `-p` option was added to `wmagent-docerk-run.sh` as well, which would trigger an image pull from `registry.cern.ch`. It is coupled with the `-v <wma.tag>` version option, so one can run a specific WMAgent version from the registry. It defaults to `latest` if skipped. 

FYI: @amaltaro